modules/nvidia_plugin/utils/openvino_patches/benchmark_app_phase1.patch

```diff
@@ -1,59 +1,63 @@
-@@ -10,3 +10,4 @@ ie_add_sample(NAME benchmark_app
+diff --git a/samples/cpp/benchmark_app/CMakeLists.txt b/samples/cpp/benchmark_app/CMakeLists.txt
+index 4cb34fe1f4..792f4579c5 100644
+--- a/samples/cpp/benchmark_app/CMakeLists.txt
++++ b/samples/cpp/benchmark_app/CMakeLists.txt
+@@ -10,7 +10,7 @@ file (GLOB HDR ${CMAKE_CURRENT_SOURCE_DIR}/*.hpp)
+ ie_add_sample(NAME ${TARGET_NAME}
+               SOURCES ${SRC}
                HEADERS ${HDR}
-               DEPENDENCIES format_reader ie_samples_utils
-               OPENCV_DEPENDENCIES core)
-+add_dependencies(benchmark_app openvino_nvidia_gpu_plugin)
-diff --git a/inference-engine/samples/benchmark_app/benchmark_app.hpp b/inference-engine/samples/benchmark_app/benchmark_app.hpp
-index af18c908e..d55baa34c 100644
---- a/inference-engine/samples/benchmark_app/benchmark_app.hpp
-+++ b/inference-engine/samples/benchmark_app/benchmark_app.hpp
-@@ -62,6 +62,9 @@ static const char enforce_bf16_message[] = "Optional. By default floating point
-                                            "                                  'true'  - enable  bfloat16 regardless of platform support\n"
-                                            "                                  'false' - disable bfloat16 regardless of platform support";
+-              DEPENDENCIES ${GFLAGS_TARGET} format_reader ie_samples_utils)
++              DEPENDENCIES ${GFLAGS_TARGET} format_reader ie_samples_utils openvino_nvidia_gpu_plugin)
+ 
+ # Required nlohmann_json dependency
+ 
+diff --git a/samples/cpp/benchmark_app/benchmark_app.hpp b/samples/cpp/benchmark_app/benchmark_app.hpp
+index 6ea383f5e4..fab5636d01 100644
+--- a/samples/cpp/benchmark_app/benchmark_app.hpp
++++ b/samples/cpp/benchmark_app/benchmark_app.hpp
+@@ -257,6 +257,9 @@ static constexpr char inference_only_message[] =
+     " To enable full mode for static models pass \"false\" value to this argument:"
+     " ex. \"-inference_only=false\".\n";
  
 +/// @brief message for optimization mode
 +static const char op_bench_mode_message[] = "Optional. Operator level benchmark on particular device";
 +
- /// @brief message for user library argument
- static const char custom_cpu_library_message[] = "Required for CPU custom layers. Absolute path to a shared library with the kernels "
-                                                  "implementations.";
-@@ -192,6 +195,9 @@ DEFINE_string(nstreams, "", infer_num_streams_message);
- /// @brief Enforces bf16 execution with bfloat16 precision on systems having this capability
- DEFINE_bool(enforcebf16, false, enforce_bf16_message);
+ /// @brief Define flag for showing help message <br>
+ DEFINE_bool(h, false, help_message);
+ 
+@@ -387,6 +390,9 @@ DEFINE_string(scale_values, "", scale_values_message);
+ /// @brief Define flag for inference only mode <br>
+ DEFINE_bool(inference_only, true, inference_only_message);
  
 +/// @brief Optimization mode for particular device
 +DEFINE_bool(op_bench, false, op_bench_mode_message);
```

Comment:
Currently we are trying to avoid introduction of device specific properties into the plugin and use `-load_config` option instead of it